nidirect_contacts/nidirect_contacts.module

```diff
@@ -19,6 +19,12 @@ function nidirect_contacts_form_views_exposed_form_alter(&$form, FormStateInterf
     // we still get to use URL parameters despite having no visible form elements.
     hide($form['sort_by']);
     hide($form['sort_order']);
+    // Add an extra class if this is a search results page.
+    if (!empty($form_state->getUserInput())) {
```

Comment:
Just a stylistic point to remove one level of indentation: you could probably refine this a little with:

```
if (!empty($form_state->getUserInput()) && !empty($form_state->getUserInput()['query_contacts_az']))
```

If the first condition fails, it won't evaluate the second as it needs both expressions to be TRUE to continue.

Reply:
Could even try dumping the return from that method into a var for checking and reduce the length of that statement by around 40 chars.

Reply:
Fair enough, have implemented those suggestions